chapters/en/module4.md

```diff
@@ -1,110 +1,184 @@
 ---
 title: 'Module 4: Getting groovy with Git and GitHub'
 description:
-  'Course introduction, summary of course learning outcomes and prerequisite validation.' 
+  'View your git history, travel back in time, deal with merge conflicts and other useful tools' 
 prev: ../../module3
 next: ../../module5
 type: chapter
-id: 3
+id: 4
 ---
 
-<exercise id="1" title="git log" type="slides,video">
 
-<slides source="module0/module0_00" shot="0" start="0:002" end="3:40">
+<exercise id="1" title="Viewing your Git history" type="slides,video">
```

Comment:
I think for slide deck titles, we should use either the -ing form of the verbs or their simple form, e.g. either "Viewing, Restoring, Reverting, Dealing, etc." or "View, Restore, Revert, Deal, etc.". I personally prefer the latter.